packages/airnode-node/src/evm/handlers/initialize-provider.test.ts

```diff
@@ -228,4 +232,31 @@ describe('initializeProvider', () => {
     expect(res).toEqual(null);
     expect(getLogsSpy).toHaveBeenCalledTimes(1);
   });
+
+  it('merges same-chain and cross-chain authorizations', () => {
+    const authorizations: AuthorizationByRequestId = {
+      '0x1': false,
+      '0x2': true,
+      '0x3': true,
+      '0x4': false,
+      '0x5': false,
+      '0x6': true,
+    };
+    const crossChainAuthorizations: AuthorizationByRequestId = {
+      '0x1': true,
+      '0x2': true,
+      '0x3': true,
```

Comment:
I guess you want to have `false` here to cover all 4 cases.

Reply:
Nice catch, thanks.